lib/apus/sent_messages.ex

```diff
@@ -13,7 +13,5 @@ defmodule Apus.SentMessages do
     Agent.update(__MODULE__, fn messages ->
       [message | messages]
     end)
-
-    message
```

Comment:
Related to the comment above on the `local_adapter`, could we change this to `{:ok, message}` to keep this clearer?